README-ZH.md

```diff
@@ -22,27 +22,27 @@ Apache ServiceComb Toolkit 是基于契约的微服务开发工具套件
 
 * 契约与代码一致性校验
 
-  校验应用的实际实现（如数据和服务API）是否与样本服务契约描述一致。
+  校验应用的实际实现（如数据和服务API）是否与样本服务契约描述一致。支持风格校验和兼容性校验,详情请见[OAS Validator](./oas-validator/README-ZH.md)
```

Comment:
I think the style check and compatibility check feature should be listed individually

Reply:
I am lazy here. Let me take a moment to think about how to organize it